data/help_templates/find-src-in-translated-help-missing-in-reference.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# print src files referenced in translated help pages, but not in the reference (en)
+
+DATA_DIR="$PWD/.."
+CHECK_FILE="help_templates/orig/index.tml"
+if [ -f "$1"/"$CHECK_FILE" ] ; then
+	DATA_DIR=$1
+elif [ -n "$1" ] || [ ! -f "$DATA_DIR"/"$CHECK_FILE" ] ; then
+	echo cannot find the data dir, set it with something like:
+	echo $0 ~/lincity-ng/data
+	exit
+fi
+cd $DATA_DIR
+
+for H in help help_templates
+do
+	echo
+	echo === directory: $H ===
+	cd $H
+	CONTENT_FILES=`find -type f -iname "*.xml" | xargs grep src | cut -d'"' -f 2 | sort | uniq`
```

Comment:
This doesn't look right for help_templates, since it doesn't contain any XML files.